diff --git frontend/src/components/account/Auth.tsx frontend/src/components/account/Auth.tsx
index 891ea30e..543bd8cf 100644
--- frontend/src/components/account/Auth.tsx
+++ frontend/src/components/account/Auth.tsx
@@ -34,6 +34,8 @@ export default function AuthToken() {
             cluster: getCluster() as string,
           })
         );
+        // On successful authentication, reload the page
+        window.location.reload();
       } else {
         setToken('');
         setShowError(true);
