diff --git frontend/src/components/account/Auth.tsx frontend/src/components/account/Auth.tsx
index b5822fd51..a1a2bdb5d 100644
--- frontend/src/components/account/Auth.tsx
+++ frontend/src/components/account/Auth.tsx
@@ -33,6 +33,7 @@ import { ApiError, testAuth } from '../../lib/k8s/apiProxy';
 import { ClusterDialog } from '../cluster/Chooser';
 import { DialogTitle } from '../common/Dialog';
 import HeadlampLink from '../common/Link';
+import { GlobalStyles } from '@mui/material';
 
 export default function AuthToken() {
   const history = useHistory();
@@ -123,6 +124,14 @@ export function PureAuthToken({
   }
 
   return (
+    <>
+    <GlobalStyles
+      styles={{
+        '.MuiDialog-container .MuiDialog-paper > h2#authtoken-dialog-title.MuiDialogTitle-root': {
+          backgroundColor: '#2c2926 !important',
+        },
+      }}
+    />
     <Box component="main">
       <ClusterDialog useCover onClose={onClose} aria-labelledby="authtoken-dialog-title">
         <DialogTitle id="authtoken-dialog-title">{title}</DialogTitle>
@@ -194,6 +203,7 @@ export function PureAuthToken({
         message={<span id="message-id">{t('translation|Error authenticating')}</span>}
       />
     </Box>
+    </>
   );
 }
 
