diff --git frontend/make-env.js frontend/make-env.js
index 1ff6fc30..22969627 100644
--- frontend/make-env.js
+++ frontend/make-env.js
@@ -9,6 +9,7 @@ const appInfo = require('../app/package.json');
 const gitVersion = execSync('git rev-parse HEAD').toString().trim();

 const envContents = {
+  GENERATE_SOURCEMAP: false,
   REACT_APP_HEADLAMP_VERSION: appInfo.version,
   REACT_APP_HEADLAMP_GIT_VERSION: gitVersion,
   REACT_APP_HEADLAMP_PRODUCT_NAME: appInfo.productName,
