diff --git frontend/src/lib/themes.ts frontend/src/lib/themes.ts
index 2ad7ad98..ffdba1d3 100644
--- frontend/src/lib/themes.ts
+++ frontend/src/lib/themes.ts
@@ -55,21 +55,21 @@ const commonRules = {
       background: '#f5f5f5',
     },
     sidebarLink: {
-      color: '#e7e7e7',
+      color: '#aea8a2', //side bar icon colors (little logos)
       main: {
         selected: {
-          color: '#000',
-          backgroundColor: '#fff200',
+          color: '#000', // color of words when clicked on
+          backgroundColor: '#d8372a', //sidebar fill color when clicking button
         },
-        color: '#fff',
+        color: '#fff', //sidebar WORDS color
       },
       selected: {
-        color: '#fff099',
+        color: '#8b8680', //sidebar sub bar fill color when clicked on
         backgroundColor: 'unset',
       },
       hover: {
         color: '#000',
-        backgroundColor: '#3B3A39',
+        backgroundColor: '#3a3633', //sidebar color when hover over a resource (ex: home)
       },
     },
     clusterChooser: {
@@ -83,12 +83,12 @@ const commonRules = {
       },
     },
     sidebarButtonInLinkArea: {
-      color: '#fff',
+      color: '#fff', //words in sidebar
       primary: {
-        background: '#605e5c',
+        background: '#605e5c', //bottom sidebar button
       },
       hover: {
-        background: '#3B3A39',
+        background: '#3B3A39', //bottom sidebar button
       },
     },
     home: {
@@ -99,18 +99,18 @@ const commonRules = {
         unknown: grey['800'],
       },
     },
-    sidebarBg: '#242424',
+    sidebarBg: '#2b2926', //sidebar color
     error: {
       main: red['800'],
       light: red['50'],
     },
     resourceToolTip: {
-      color: 'rgba(0, 0, 0, 0.87)',
+      color: '#fff',
     },
     normalEventBg: '#F0F0F0',
     chartStyles: {
       defaultFillColor: grey['300'],
-      labelColor: '#000',
+      labelColor: '#fff',
     },
     metadataBgColor: '#f3f2f1',
     headerStyle: {
@@ -146,7 +146,7 @@ const commonRules = {
     },
   },
   typography: {
-    fontFamily: ['Overpass', 'sans-serif'].join(', '),
+    fontFamily: ['Oracle Sans', 'sans-serif'].join(', '),
     h1: {
       fontWeight: 700,
       fontSize: '1.87rem',
